admin_tests/users/test_views.py

```diff
@@ -629,6 +629,16 @@ def test_search_user_by_name(self):
         nt.assert_equal(response.status_code, 302)
         nt.assert_equal(self.view.success_url, '/users/search/Hardy/')
 
+    def test_search_user_by_name_with_punction(self):
+        form_data = {
+            'name': 'Dr. Sportello'
```

Comment:
You should go ahead and put a comma in here as well. 'Dr. Sportello, DDS.' for example.